translate/storage/php.py

```diff
@@ -54,13 +54,140 @@
 `String type <http://www.php.net/language.types.string>`_.
 """
 
-import logging
+from __future__ import unicode_literals
+
 import re
+
 import six
 
+from phply.phpparse import make_parser
+from phply.phplex import lexer, FilteredLexer, full_lexer, unparsed
+from phply.phpast import (InlineHTML, FunctionCall, Assignment, ArrayOffset,
+                          Variable, Array, ArrayElement, Return, BinaryOp,
+                          Node)
```

Comment:
These imports still need some changes to conform to http://docs.translatehouse.org/projects/translate-toolkit/en/latest/developers/styleguide.html#imports